tools/ui-components/src/declarations.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.css';
```

Comment:
This is needed as TypeScript wasn't happy with the import statement.

Ref: https://stackoverflow.com/a/68011107

<img width="876" alt="Screen Shot 2022-01-30 at 22 58 11" src="https://user-images.githubusercontent.com/25715018/151707397-1b790dd3-4618-435e-9100-1d0dd2ad62e2.png">